vault/core.go

```diff
@@ -884,7 +890,11 @@ func (c *Core) Initialize(config *SealConfig) (*InitResult, error) {
 	}
 
 	if len(config.PGPKeys) > 0 {
-		_, encryptedShares, err := pgpkeys.EncryptShares(results.SecretShares, config.PGPKeys)
+		hexEncodedShares := make([][]byte, len(results.SecretShares))
+		for i, _ := range results.SecretShares {
+			hexEncodedShares[i] = []byte(hex.EncodeToString(results.SecretShares[i]))
```

Comment:
To clarify: they are hex encoded, but the encryption function returns binary data, so we need to hex encode that. This hex data is then decoded, that binary value is decrypted, and the resulting value is the hex-encoded key. It should have been base64 for transit, except...it wasn't. Maybe we'll change that at some point.